tests/test_compare_pyuvsim.py

```diff
@@ -5,20 +5,26 @@
 from astropy.coordinates import EarthLocation, Latitude, Longitude
 from astropy.time import Time
 from astropy.units import Quantity
+from pathlib import Path
 from pyradiosky import SkyModel
+from pyuvdata import UVBeam
 from pyuvsim import AnalyticBeam, simsetup, uvsim
 from pyuvsim.telescope import BeamList
 
+from vis_cpu import __path__ as vis_cpu_path
```

Comment:
OK, I think you can get past this error just by using the path to _this file_ instead of the installed vis-cpu. 

I.e. `beam_file = Path(__file__).parent / 'data/NF_xxxx'`

Reply:
Nice idea. Let's see if it works!